setup.py

```diff
@@ -7,7 +7,7 @@
 from setuptools import find_packages, setup
 
 setup(
-    name="segment_anything",
+    name="segment_anything_hq",
```

Comment:
I believe this is the crucial fix for the _"Generating metadata for package segment-anything-hq produced metadata for project name segment-anything"_ error.

See also https://github.com/pypa/pipenv/issues/5445#issuecomment-1297014393